SimPEG/EM/FDEM/SrcFDEM.py

```diff
@@ -509,3 +557,150 @@ def _srcFct(self, obsLoc, component):
                                            mu=self.mu, radius=self.radius,
                                            orientation=self.orientation)
 
+class PrimSecSigma(BaseSrc):
+
+    def __init__(self, rxList, freq, sigBack, ePrimary, **kwargs):
+        self.sigBack = sigBack
+
+        BaseSrc.__init__(self, rxList, freq=freq, _ePrimary=ePrimary, **kwargs)
+
+    def s_e(self, prob):
+        return (prob.MeSigma -  prob.mesh.getEdgeInnerProduct(self.sigBack)) * self.ePrimary(prob)
+
+    def s_eDeriv(self, prob, v, adjoint=False):
+        if adjoint:
+            return prob.MeSigmaDeriv(self.ePrimary(prob)).T * v
+        return prob.MeSigmaDeriv(self.ePrimary(prob)) * v
+
+
+class PrimSecMappedSigma(BaseSrc):
```

Comment:
- [ ] this needs to be generalized (or at least user warnings added). Currently functionality is strictly simulating H-J on the primary, E-B on the secondary. 
- [ ] also - tests from the simpegEM paper can be brought in